DESCRIPTION

```diff
@@ -24,7 +24,7 @@ Depends:
     R (>= 3.0.0)
 Imports:
     farver,
-    ggplot2 (>= 3.3.0),
+    ggplot2 (>= 3.5.2),
```

Comment:
I don't think there's a 'real' reason to require a recent version?

```suggestion
    ggplot2 (>= 3.3.0),
```

Reply:
I've used `is_theme_element()` somewhere to test if an element is blank in a S3/S7 agnostic way, but that'd require 3.5.2. I guess you could work around that, but it can be gnarly.